js/src/forum/components/MoreDiscussions.tsx

```diff
@@ -55,6 +57,9 @@ export default class MoreDiscussions<CustomAttrs> extends Component {
           limit: this.limit,
         },
         sort: this.filterBy,
+        filter: {
+          q: this.isRelated ? app.current.data.discussion.title : null,
```

Comment:
Look like it is `app.current.data.discussion.title()` not `app.current.data.discussion.title`

Reply:
Solved

Reply:
Thank you 👍🏼

Reply:
Will this also be resolved https://github.com/datlechin/flarum-more-discussions/issues/7?